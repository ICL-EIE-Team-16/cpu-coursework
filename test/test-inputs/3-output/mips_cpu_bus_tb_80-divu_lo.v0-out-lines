REG v0: OUT: 000e1035
