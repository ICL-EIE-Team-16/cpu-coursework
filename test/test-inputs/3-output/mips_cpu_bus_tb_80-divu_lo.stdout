Loading from test-inputs/1-hex/80-divu_lo.hex.txt
WARNING: ../rtl/mips_cpu/simple_memory.v:33: $readmemh(test-inputs/1-hex/80-divu_lo.hex.txt): Not enough words in the file for the requested range [0:1023].
VCD info: dumpfile waves.vcd opened for output.
num: 10000000000000000000000000000000: 
shifted num: 00000000000000000000100000000000: 
REGFile : OUT: $zero,$at,$v0,$v1,$a0,$a1,$a2,$a3,$t0,$t1,$t2,$t3,$t4,$t5,$t6,$t7,$s0,$s1,$s2,$s3,$s4,$s5,$s6,$s7,$t8,$t9,$k0,$k1,$gp,$sp,$s8,$ra
r=xxxxxxxx
r=xxxxxxxx
address: bfc00000
r=00000000
r=ffffffff
address: ffffffff
address: ffffffff
address: bfc00004
r=ffff0000
address: ffff0000
address: ffff0000
r=00000000
address: bfc00008
r=fffe0000
r=ffff1234
address: ffff1234
address: ffff1234
r=ffff2468
address: bfc0000c
r=00001234
address: 00001234
address: 00001234
address: bfc00010
r=00001234
r=00001234
address: 00001234
address: 00001234
address: bfc00014
r=000e1035
address: 000e1035
address: 000e1035
address: bfc00018
r=000e1035
address: 000e1035
address: 000e1035
address: bfc0001c
REG v0: OUT: 000e1035
